dind/run.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+thisDir=$(dirname "$0")
+. "${thisDir}/config"
+. "${thisDir}/lib.bash"
+
+for index in ${nodes[*]}
+do
+    mountUp "${index}"
+done
+
+docker-compose start
```

Comment:
if you use build context in docker compose I guess it would be nicer to use 
``docker-compose up -d --build``

Reply:
purpose was to up/build in bootstrap.bash (invoke and run), in travis only bootstrap.bash will be used (and then tests run). start.bash purpose is to run already bootstrapped nodes in dev environnement. ok for you ? (stop.bash to stop nodes in dev env)